www/js/controller.js

```diff
@@ -1000,7 +1000,7 @@ define([
 					}));
 				}
 
-				if(parseObject.priorError){ //priorError specifically indicates if there is an error where in a non dynamic node/variable is used inside prior function
+				if(parseObject.error){ //error specifically indicates if there is an error where in a non dynamic node/variable is used inside prior function
```

Comment:
This comment is no longer true, I think.   Now it indicates other types of errors too.  (That's one of the reasons you renamed it I think.)  Please update the comment as needed.

Reply:
This comment is true, I updaed it for different reason. Earlier I am using success and priorError for the same variable. So I gave it one generic name. It is still the marked for prior error only.